aws-lambda-java-events/src/main/java/com/amazonaws/services/lambda/runtime/events/APIGatewayProxyRequestEvent.java

```diff
@@ -267,6 +269,19 @@ public ProxyRequestContext withPath(String path) {
             return this;
         }
 
+        public Map<String, String> getAuthorizer() {
+            return authorizer;
+        }
+
+        public void setAuthorizer(Map<String, String> authorizer) {
```

Comment:
From https://github.com/aws/aws-lambda-dotnet/issues/98, `"authorizer"` may contain more than just strings. To be truly generic, it might be that a `APIGatewayAuthorizedProxyEvent<T>` needs to be added.